Epsilon.Host.Frontend/src/views/DashboardView.vue

```diff
@@ -39,17 +34,29 @@ const store = useStore()
 onMounted(() => {
     api?.filter.accessibleStudentsList().then((r: HttpResponse<User[]>) => {
         store.commit("setUsers", r.data)
-        store.commit("setCurrentUser", store.state.users[0])
+        store.commit(
+            "setCurrentUser",
+            store.state.users.find(
+                (u: User) => u._id === import.meta.env.VITE_USER_ID
+            )
+        )
     })
     api?.learning
         .domainDetail("hbo-i-2018")
         .then((r) => store.commit("setDomain", r.data))
 
+    api?.learning
+        .domainDetail("pd-2020-bsc")
+        .then((r) => store.commit("setPersonalDevelopment", r.data))
+
     api?.learning
         .outcomesList({
-            studentId: "20592",
+            studentId: import.meta.env.VITE_USER_ID ?? "00000",
```

Comment:
To add onto this, we shouldn't use default values like this. Like you say, Canvas will return an unauthorized error, but that does not indicate that the user id has not been set, which is what you actually want to know and see. Maybe throw an exception instead, or make it clear that the user id has not been set.